Event/ConnectorCalledEvent.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Integrated package.
+ *
+ * (c) e-Active B.V. <integrated@e-active.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Integrated\Bundle\ChannelBundle\Event;
+
+use Integrated\Common\Channel\Connector\Config\OptionsInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+class ConnectorCalledEvent extends Event
```

Comment:
This name does not describe what this event class does. A better name would probably GetResponseConfigEvent.